docs/api/service.md

```diff
@@ -38,12 +38,12 @@ The event and its data must be defined in the [service's definition file](../gui
 #### EmitEventRequest
 The request's data for the `EmitEvent` API.
 
-**Example:**
+Example:
 ```json
 {
-  "token":     "__SERVICE_TOKEN_FROM_ENV__",
-  "eventKey":  "__EVENT_KEY__",
-  "eventData": "{\"foo\":\"hello\",\"bar\":false}"
+"token":     "__SERVICE_TOKEN_FROM_ENV__",
```

Comment:
take care there is some problems with the generation of the doc, all the json are flatten and the example is not in bold anymore

https://deploy-preview-541--mesg-docs.netlify.com/api/core.html#getservice

Reply:
Do you know how to solve this without manually editing the docs? Running _build-proto.sh_ doesn't correct the docs.

Reply:
What is your version?

Reply:
For protoc I have `libprotoc 3.6.1` (I think this one is the same, there is not much versions)
For protoc-gen-go I cannot have the version but the commit hash is `b27b920f9e71b439b873b17bf99f56467623814a` this one should be the problem